include/picongpu/particles/collision/relativistic/RelativisticCollision.hpp

```diff
@@ -182,7 +182,7 @@ namespace picongpu
 
                         // (12) in [Perez 2012]
                         float3_COLL finalVec;
-                        float_COLL const pAbs = math::sqrt(pmacc::math::abs2(p));
+                        float_COLL const pAbs = math::sqrt(pmacc::math::l2norm2(p));
```

Comment:
```suggestion
                        float_COLL const pNorm2 = math::sqrt(pmacc::math::l2norm2(p));
```
see also renamed variable name below